<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>944</width>
    <height>793</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1">
     <widget class="QTimeOrganizerWidget" name="timeOrganaizerWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="groupBoxSettings">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Widget settings</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="5" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Scroll mode</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="comboBoxScrollMode">
         <item>
          <property name="text">
           <string>Scroll hours</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ongoing scroll</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="spinBoxHours">
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="minimum">
          <number>-1</number>
         </property>
         <property name="maximum">
          <number>49</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Mninmum item height: </string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Hours displayed</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Selected range</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLineEdit" name="lineEditSelectedRange"/>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="spinBoxMinItemHeight">
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="maximum">
          <number>200</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QCheckBox" name="checkBoxShowHint">
         <property name="text">
          <string>Show hints</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QCheckBox" name="checkBoxTracking">
         <property name="text">
          <string>Tracking</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="pushButtonHideHeader">
         <property name="text">
          <string>Hide header</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButtonShowHeader">
         <property name="text">
          <string>Show header</string>
         </property>
        </widget>
       </item>
       <item row="13" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Date - time</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="0">
           <widget class="QPushButton" name="pushButtonSetDate">
            <property name="text">
             <string>Set date</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDateEdit" name="dateEdit">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="spinBoxHour">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="maximum">
             <number>23</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="pushButtonSetHour">
            <property name="text">
             <string>Set hour</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="10" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Selection</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0" colspan="2">
           <widget class="QDateTimeEdit" name="dateTimeEditRangeBegin">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0" rowspan="2" colspan="2">
           <widget class="QDateTimeEdit" name="dateTimeEditRangeEnd">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="pushButtonSetRange">
            <property name="text">
             <string>Set range</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="pushButtonClearSelection">
            <property name="text">
             <string>Clear selection</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="11" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Busy range</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="2" column="1">
           <widget class="QPushButton" name="pushButtonClearBusyRange">
            <property name="text">
             <string>Clear busy range</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDateTimeEdit" name="dateTimeEditBusyRangeEnd"/>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="pushButtonSetBusyRange">
            <property name="text">
             <string>Set busy range</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QDateTimeEdit" name="dateTimeEditBusyRangeBegin"/>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QLineEdit" name="lineEditBusyText">
            <property name="text">
             <string>Some text</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="14" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Grid settings</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Grid separation by</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="checkBoxGrid">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Enable grid</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="checkBoxDrawGrid">
            <property name="text">
             <string>Draw grid</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="comboBoxGridSep">
            <item>
             <property name="text">
              <string>Hour</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>30 minutes</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>15 minutes</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>10 minutes</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>5 minutes</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Minute</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>30 seconds</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>944</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QTimeOrganizerWidget</class>
   <extends>QWidget</extends>
   <header location="global">qtimeorganizerwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
